packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -209,7 +214,7 @@ export const ChatMessage: ComponentWithAs<'div', ChatMessageProps> &
   const { setStart, setEnd } = useTelemetry(ChatMessage.displayName, context.telemetry);
   setStart();
 
-  const parentAttached = useContextSelector(ChatItemContext, v => v.attached);
+  const { attached: parentAttached, compact } = useContextSelector(ChatItemContext, v => v);
```

Comment:
This will cause unnecessary rerenders, please use multiple context selectors so that you only fetch primitives or memoized values from context

```suggestion
  conat parentAttached = useContextSelector(ChatItemContext, v => v.attached);
  const compact  = useContextSelector(ChatItemContext, v => v.compact);
```